web/src/hooks/useMarkets.ts

```diff
@@ -73,8 +80,32 @@ export const useGraphMarkets = (
           where["creator"] = creator;
         }
 
-        const { markets } = await getSdk(client).GetMarkets({ where, orderBy, orderDirection: OrderDirection.Desc });
+        let markets: {
+          __typename?: "Market";
+          id: string;
+          factory: `0x${string}`;
+          creator: `0x${string}`;
+        }[] = [];
+
+        let lastId = null;
+
+        // try to fetch all markets on subgraph
+        while (true) {
+          const { markets: currentMarkets } = await getSdk(client).GetMarkets({
+            where: { ...where, ...(lastId && { id_gt: lastId }) },
```

Comment:
it's working now because we have less than 1000 markets, but for this to work we also need to sort by `id`, otherwise it will not return all the markets (example: the lastId returned in the first fetch is the max id, if we have more markets they will not be returned in the next fetch)